store/snap_remote_repo_test.go

```diff
@@ -163,7 +163,7 @@ func (t *remoteRepoTestSuite) TestUbuntuStoreRepositoryHeaders(c *C) {
 
 	t.store.applyUbuntuStoreHeaders(req, "", nil)
 
-	c.Assert(req.Header.Get("X-Ubuntu-Release"), Equals, release.String())
+	c.Assert(req.Header.Get("X-Ubuntu-Release"), Equals, release.Get().Series)
```

Comment:
Can we please have a string here? Right now it is testing implementation against implementation, which will pass even if we break it.